docs/proposals/karmadactl/Karmadactl Supports Configuration File Method for Production-Grade Installation and Deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Karmadactl Supports Configuration File Method for Production-Grade Installation and Deployment
+authors:
+  - "@tiansuo114"
+  - "@liangyuanpeng"
```

Comment:
```suggestion
  - "@tiansuo114"
```
You can put me into reviewer :)

Reply:
Ok, I've noticed that I might also need to add some specific data structure content, which I'll do in the next revision